src/app/components/grid/grid.component.ts

```diff
@@ -8,11 +9,17 @@ import { GridsterConfig, GridsterItem }  from 'angular-gridster2';
 })
 export class GridComponent implements OnInit {
   options: GridsterConfig;
-  dashboard: Array<GridsterItem>;
+  windows:Array<Window>;
 
   constructor() { }
 
   ngOnInit() {
+    //need to inject service here and subscribe to changes 
+    this.windows = [
```

Comment:
Will need to have this component read from the data service to load in the appropriate windows. This will probably be done in issue #10.